ehitchhiking-ui/src/app/shared/modals/profile-modal/car-info-form/car-info-form.component.ts

```diff
@@ -0,0 +1,55 @@
+import {Component, Input, OnInit, Output, EventEmitter} from '@angular/core';
+import {FormGroup, FormBuilder, Validators} from '@angular/forms';
+import {Car} from '../../../models/car';
+@Component({
+	selector: 'app-car-info-form',
+	templateUrl: './car-info-form.component.html',
+	styleUrls: ['./car-info-form.component.sass'],
+})
+export class CarInfoFormComponent implements OnInit {
+	@Input() carInfoForm: FormGroup;
+	@Input() addCarMod: boolean;
+	@Input() carIndex: number;
+	@Output() onChangeCarInfo = new EventEmitter<Car>();
+	addCarForm: FormGroup;
+
+	constructor(private formBuilder: FormBuilder) {}
+
+	ngOnInit() {
+		if (this.addCarMod) {
```

Comment:
I think that shouldn't happen, I mean falsy condition

Reply:
right, i think so too, done